
Trivial compile fix by removing fields gone in 2.6.11.

  - Rene Rebe <rene@exactcode.de>

diff -ur acx100-0.2.0pre8_plus_fixes_45/src/acx100_usb.c acx100-0.2.0pre8_plus_fixes_45-fixed/src/acx100_usb.c
--- acx100-0.2.0pre8_plus_fixes_45/src/acx100_usb.c	2005-01-30 00:45:33.000000000 +0100
+++ acx100-0.2.0pre8_plus_fixes_45-fixed/src/acx100_usb.c	2005-03-05 22:26:29.628244048 +0100
@@ -1802,12 +1802,12 @@
   printk(KERN_INFO "  ttport: %d\n",(unsigned int)(usbdev->ttport));
   printk(KERN_INFO "  toggle[0]: 0x%X  toggle[1]: 0x%X\n",(unsigned int)(usbdev->toggle[0]),(unsigned int)(usbdev->toggle[1]));
   /* printk(KERN_INFO "  halted[0]: 0x%X  halted[1]: 0x%X\n",usbdev->halted[0],usbdev->halted[1]); halted removed in 2.6.9-rc1 */
-  printk(KERN_INFO "  epmaxpacketin: ");
+/*  printk(KERN_INFO "  epmaxpacketin: ");
   for (i=0;i<16;i++) printk("%d ",usbdev->epmaxpacketin[i]);
   printk("\n");
   printk(KERN_INFO "  epmaxpacketout: ");
   for (i=0;i<16;i++) printk("%d ",usbdev->epmaxpacketout[i]);
-  printk("\n");
+  printk("\n"); */
   printk(KERN_INFO "  parent: 0x%X\n",(unsigned int)(usbdev->parent));
   printk(KERN_INFO "  bus: 0x%X\n",(unsigned int)(usbdev->bus));
 #if NO_DATATYPE
